### ISSUE TITLE
[Bug] Will Not Compile On Node 12

### ISSUE THREAD

**Issue by wozeparrot** [JsSucks/BetterDiscordApp#368]
- [x] Used appropriate template for the issue type
- [x] Searched both open and closed issues for duplicates of this issue
- [x] Title adequately and _concisely_ reflects the feature or the bug

**Describe the bug**
keytar and node-sass will not compile on node 12. Also tried node 13, did not work there either.
Might not be related to node, as make is the one that is throwing the error.

**To Reproduce**
use node 12 and run `npm install`.

**Expected behavior**
Expected to be able to compile keytar and node-sass.

**Screenshots**
None

**System information**
GNU/Linux 5.4.0
Discord PTB

**Additional context**
None

**Are you willing and able to fix this?**
No. Gotta learn more js.

**Comment by wozeparrot** [comment]
Seems that the Travis build is also failing due to the same issue.

**Comment by 0x57e11a** [comment]
This ever gonna get fixed? Really havin some problems here

**Comment by zerebos** [comment]
I would say unlikely as this hasn't been essentially abandoned. It was also was not meant for end-users yet (while it was actively worked), only for the developers. And this was one of those errors that was easy to get around for us so we never bothered.

**Comment by 0x57e11a** [comment]
Yeah, I'd just like to be able to have Node v12 for my personal NJS projects, and I assumr the fix is downgrading

**Comment by zerebos** [comment]
You can continue to use Node 12. As I said this is essentially abandoned so you don't need to worry about building this repo.
